src/Fmt_odoc.ml

```diff
@@ -145,7 +145,7 @@ let rec fmt_inline_element : inline_element -> Fmt.t = function
       escape $ str_normalized w
   | `Code_span s ->
       let s = escape_brackets s in
-      hovbox 0 (wrap "[" "]" (str s))
+      hovbox 0 (wrap "[" "]" (str_normalized s))
```

Comment:
The CI is fixed if you use `~escape:false`, is it necessary to escape all these characters?

Reply:
It's not necessary to escape all of them, only `[` and `]`. Also `str_normalized` will remove consecutive whitespaces, which may not be what we want.

Reply:
I changed to only escape brackets and to keep consecutive whitespaces. The CI is fixed.

Reply:
I'm not convinced regarding keeping consecutive whitespaces, I think if someone would want to keep the spaces he would use the verbatim blocks instead.

Reply:
I reverted that part.